alena_automation/XPathExercises - Alena2.csv

```diff
@@ -0,0 +1,34 @@
+Xpath Tasks,Authors.xml,Authors2.xml (with namespaces)
+Normal:,,
+all elements,/authors,
+"all ""author"" elements",//author,
+"the root element ""authors"" - 2 variants","/* ,  authors, child::authors",
+"the same as 3,  but this time without using the name of the element",/*,
+all elements enclosed by elements  /authors/author,/authors/author,
+"all ""name"" elements (without using names of parent elements) ",//name,
+all elements which have 1 ancestors,,
+"all ""period"" attributes of ""author"" elements - 2 variants","//author[@period] , ",
+"all ""author"" elements with a ""period"" attribute. ",//author[@period],
+"the fourth ""author"" element. ",//author[4],
+"the ""name"" node of the fourth author",//author[4]/name,
+the name of the fourth author (text),//author[4]/name[text()],
+all elements that have any attributes,//*[@*],
+all elements without any attributes,/*/*/*[not(@*)],
```

Comment:
//\*[not(@*)]

Reply:
if paste //*[not(@*)]   i see the following returned:

Element='<authors>
  <author born="1802" died="1885">
      <name idd="VH">Victor Hugo</name>
      <nationality>French</nationality>
  </author>
  <author born="-496" died="-406" period="classical">
      <name idd="SO">Sophocles</name>
      <nationality>Greek</nationality>
  </author>
  <author born="1828" died="1910">
      <name idd="LT">Leo Tolstoy</name>
      <nationality>Russian</nationality>
  </author>
  <author born="1799" died="1837">
      <name idd="AP">Alexander Pushkin</name>
      <nationality>Russian</nationality>
  </author>
  <author born="-427" died="-347" period="classical">
      <name idd="PL">Plato</name>
      <nationality>Greek</nationality>
  </author>
</authors>'
Element='<nationality>French</nationality>'
Element='<nationality>Greek</nationality>'
Element='<nationality>Russian</nationality>'
Element='<nationality>Russian</nationality>'
Element='<nationality>Greek</nationality>'